adcm_aio_client/core/objects/cm.py

```diff
@@ -192,6 +192,19 @@ def components(self: Self) -> "ComponentsNode":
 class ServicesNode(PaginatedChildAccessor[Cluster, Service, None]):
     class_type = Service
 
+    async def get_status(self: Self) -> ADCMEntityStatus:
+        response = await self._requester.get(*self._parent.get_own_path())
+        return ADCMEntityStatus(response.as_dict()["status"])
+
+    async def delete(self: Self) -> None:
+        await self._requester.delete(*self._path)
+
+    async def turn_maintenance_mode_on(self: Self) -> None:
+        await self._requester.post(*self._path, "maintenance-mode", data={"maintenanceMode": MaintenanceModeStatus.ON})
+
+    async def turn_maintenance_mode_off(self: Self) -> None:
+        await self._requester.post(*self._path, "maintenanceMode", data={"maintenanceMode": MaintenanceModeStatus.OFF})
```

Comment:
how you can do any of these operations on Node?

it's object's methods at  best (and most of them are implemented, I believe)

---

`add` here is missing

Reply:
What is Node supposed for then? add is not listed in requirements here https://wiki.adsw.io/books/adcm/page/adcm-aio-client#bkmrk-host-attributes%3A-id%3B-1